pihole-google.txt

```diff
@@ -7452,4 +7462,345 @@ www.tf-enterprise-devel.gcp.fitbit.com
 www.vps.store-au.fitbit.com
 www.wiki.fitbit.com
 www.windchill.fitbit.com
-wwwinternal.fitbit.com
\ No newline at end of file
+wwwinternal.fitbit.com
+ 1e100.net       # Google Cookies
```

Comment:
Can you please remove all comments? The automatic will probably fail because of the extra comments. Thank you!

Reply:
I'm trying but it won't let me. if I close this suggestion, and reopen it, are things saved or removed?

Reply:
If you resolve and reopen this suggestion/review. No changes will be made to your files.